src/mudlet.cpp

```diff
@@ -3717,8 +3722,6 @@ void mudlet::doAutoLogin(const QString& profile_name)
     // save the setting is on profile loading:
     pHost->mTelnet.setEncoding(readProfileData(profile_name, QLatin1String("encoding")), false);
 
-    // For the first real host created the getHostCount() will return 2 because
-    // there is already a "default_host"
     signal_hostCreated(pHost, mHostManager.getHostCount());
```

Comment:
If you haven't already done so, may I suggest checking any `slot`s connected to this `signal` to ensure that they respond correctly given that the number passed will be one less now...?

Reply:
Yes, the only place it is used in is the settings window, and that works OK.

Also, this is a strange way to emit a signal...

Reply:
🤔 Humm, yeah - where has the `emit` gone?

Reply:
It was like that from the beginning https://github.com/Mudlet/Mudlet/commit/92f61bb7d5576c6c429a0a491294581c07012676#diff-84e2883dc57a4e7b034fbac3fd4668b5R2633 - I'll fix it.